key-terms.md

```diff
@@ -25,23 +25,25 @@ The term **closed functionality**, as used in WCAG2ICT, has the meaning below:
 
 <DL><DT>closed functionality (as used in WCAG2ICT)</DT><DD>
 
-a property or characteristic that prevents users from attaching, installing, or using [assistive technology](#dfn-assistive-technologies)</DD></DL>
```

Comment:
The current definition is sufficient and understandable.  Modification of the definition is will be:
* Different than EN301 549 Section 3.1 and [US S508](https://www.access-board.gov/ict/#E103.4), 
* Already sufficiently address in the following note, examples (below the current definition) and other areas of WCAG2ICT.

Reply:
The proposed changes to the definition of "closed functionality" have been backed out due to the TF resolution on 6 June.